### PR TITLE
Fixes for history view

### PR DESCRIPTION
**Pull request by cghislai**

### CODE REVIEW

File: comptoir-ws/src/main/java/be/valuya/comptoir/ws/rest/control/EmployeeResource.java [hunk 1]
```diff
@@ -93,7 +93,6 @@ public WsEmployee getEmployee(@PathParam("id") long id) {
         return wsEmployees;
     }
 
-    @Valid
```
Comment (cghislai) [inline comptoir-ws/src/main/java/be/valuya/comptoir/ws/rest/control/EmployeeResource.java]: Glassfish complains that this is not allowed on method that returns void

File: comptoir-ws/src/main/java/be/valuya/comptoir/ws/rest/control/SaleResource.java [hunk 2]
```diff
@@ -141,8 +141,6 @@ public WsSaleRef closeSale(@PathParam("id") long id) {
     @Path("{id}/payed")
     public BigDecimal getSaleTotalPayed(@PathParam("id") long id) {
         Sale sale = saleService.findSaleById(id);
-        saleStateChecker.checkState(sale, false); // TODO: replace with bean validation
-
         return saleService.getSaleTotalPayed(sale);
```
Comment (cghislai) [inline comptoir-ws/src/main/java/be/valuya/comptoir/ws/rest/control/SaleResource.java]: For simplicity, I call this as well when displaying closed sales

Comment (ymajoros) [inline comptoir-ws/src/main/java/be/valuya/comptoir/ws/rest/control/SaleResource.java]: Why, closed sales are totally payed anyway? Do you display payments for
closed sales somewhere?

Comment (cghislai) [inline comptoir-ws/src/main/java/be/valuya/comptoir/ws/rest/control/SaleResource.java]: yes, when displaying sales from the history

Comment (ymajoros) [inline comptoir-ws/src/main/java/be/valuya/comptoir/ws/rest/control/SaleResource.java]: Let's remove that then ;-)

Le mer. 26 août 2015 à 13:57, cghislai <notifications@github.com> a écrit :

> In
> comptoir-ws/src/main/java/be/valuya/comptoir/ws/rest/control/SaleResource.java
> <https://github.com/Valuya/comptoir/pull/56#discussion_r37973150>:
>
> > @@ -141,8 +141,6 @@ public WsSaleRef closeSale(@PathParam("id") long id) {
> >      @Path("{id}/payed")
> >      public BigDecimal getSaleTotalPayed(@PathParam("id") long id) {
> >          Sale sale = saleService.findSaleById(id);
> > -        saleStateChecker.checkState(sale, false); // TODO: replace with bean validation
> > -
> >          return saleService.getSaleTotalPayed(sale);
>
> yes, when displaying sales from the history
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Valuya/comptoir/pull/56/files#r37973150>.
>
-- 
Yannick Majoros
Valuya sprl

**Pull request merged**